cypress/integration/veiledergrupper_spec.js

```diff
@@ -155,10 +155,11 @@ describe('Rediger filtervalg', () => {
             .should('be.visible')
             .contains('Gruppen er lagret');
     });
+
     it('Sjekk at det er ett filtervalg', () => {
         cy.getByTestId('filtreringlabel')
             .should('have.length', 1)
-            .contains(aasen);
+            .contains(andersen);
```

Comment:
![image](https://user-images.githubusercontent.com/21334782/97988868-f1677800-1ddd-11eb-9f67-de0becfe2dd4.png)
Should be Aasen!